understanding/understanding-metadata.html

```diff
@@ -150,13 +127,10 @@ <h2>Metadata Resources</h2>
                      			
                   </ul>
                   			
-                  <p>There are some tools available to provide resource descriptions, or they can be provided
-                     manually. The more easily the metadata can be created and collected at point of creation
-                     of a resource or at point of publication, the more efficient the process and the more
-                     likely it is to take place. 
+                  <p>リソースの説明を提供するツールがいくつか入手可能である。あるいは、その説明は人的に提供することが可能である。メタデータの作成及びリソースの作成又は発行時点でのメタデータの収集がもっと容易になればなるほど、プロセスはより効果的になり、より多く使われるようになるはずである。
```

Comment:
冒頭部分はなんか変なのでは。例えばこうでしょうか。

```suggestion
                  <p>リソースの説明を提供するためにいくつかのツールを使うことができ、あるいは、その説明を手動で提供することも可能である。メタデータの作成及びリソースの作成又は発行時点でのメタデータの収集がもっと容易になればなるほど、プロセスはより効果的になり、より多く使われるようになるはずである。
```

Reply:
もう少し自然な日本語にするとすれば、こうですかね？

リソースの説明を提供するために利用できるツールがいくつかあるが、手動で提供することもできる。